_assets/ci/Jenkinsfile.desktop

```diff
@@ -112,11 +112,13 @@ pipeline {
     }
     stage('Cleanup') {
```

Comment:
Wait, why is this in some kind of special `Cleanup` stage when there's a dedicated `post { cleanip {} }` stage?